server/src/pages/shot/time-diff.js

```diff
@@ -24,6 +24,12 @@ exports.TimeDiff = class TimeDiff extends React.Component {
     }
   }
 
```

Comment:
> I wonder if we should consider using some library

It would be cool if we could either contribute back to that library, or export translations out of mozilla-central into a helpful localized string toolkit (common stuff like "Download" or "Cancel" in many languages)